Dockerfile

```diff
@@ -5,3 +5,11 @@ RUN pip install numpy pandas flask pygal smalluuid zipstream python-dateutil git
 WORKDIR /app
 
 ADD . /app
+
+RUN pip install -r dev-requirements.txt
+RUN pip install -r tox-requirements.txt
+RUN pip install -e .
+
+RUN apt-get install -y gnupg && curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get install -y nodejs
+RUN cd mlflow/server/js && npm run build
```

Comment:
I ran this and it didn't work for me, do we need to run an `npm install` before this?

Reply:
Yeah, good catch.  Thanks.